.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ios"]
+	path = ios
+	url = https://github.com/nimblehq/ios-templates.git
```

Comment:
I prefer to set up it by version, because a new update from `main` could break this kmm template  🙏

Reply:
When I initiated the submodule, it was tied to the commit hash  `8336744cc7f66001c9ed24018ff2f545ff9a3c5a`. You can see the commit hash of the `ios` module right below. 

![Screenshot 2023-08-08 at 18 12 53](https://github.com/nimblehq/kmm-templates/assets/22606906/79d9e81c-947d-4ec8-9f80-c2ef8a7e8b21)